blog/2025-02-21/index.md

```diff
@@ -0,0 +1,123 @@
+
+## Reproducibility & Auditability
+
+Once an app has been approved and passes initial tests, it is built using the open source and publicly-available flatpak-builder utility from the approved public manifest, on Flathub’s infrastructure, and without network access. Sources for the app are validated against the documented checksums, and the build fails if they do not match.
+
+For further auditability, we insert the git short rev into the Flatpak build subject so anyone can confirm which commit of the manifest repo a specific build was built from. The build itself is signed by Flathub’s key, and Flatpak/OSTree verify these signatures when installing and updating apps.
+
+We mirror the exact sources each app is built against in case the original source goes down or there is some other issue, and anyone can build the Flatpak back from those mirrored source to reproduce or audit the build. The manifest used to build the app is distributed to every user in the app’s sandbox at `/app/manifest.json`, which can be inspected and used to rebuild the app exactly as it was built by Flathub.
+
+## Verification
+
+Apps can be [verified](https://docs.flathub.org/docs/for-app-authors/verification) on Flathub; this process confirms that an app is published by the original developer or an authorized party by proving ownership of the app ID. While all apps are held to the same high standards of safety and review on Flathub, this extra layer helps users confirm that the app they are getting is _also_ provided or authorized by its developer.
+
+![Verified checkmark](verified.png)
+
+Over half of the apps on Flathub so far are verified, with the number regularly increasing.
+
+## App Store Clients
+
+Once an app is developed, submitted, tested, approved, built, and distributed, it appears in app store clients like Flathub.org, KDE Discover, GNOME Software, and elementary AppCenter—as well as the Flatpak CLI. While exact implementations vary and the presentation is up to the specific app store client, generally each will show:
+
+- Static permissions and their impact on safety
+- Open Age Rating Service rating and details
+- If an app uses outdated runtimes
+- Release notes for each release
+- If static permissions increase between releases
+
+Flathub.org and GNOME Software also display the app’s verified status.
+
+## Updates
+
+Once an app is accepted onto Flathub, it’s not just the wild west; there are still a number of safety protections built into the flow:
+
+- **Flathub maintains ownership over the manifest repo**, while app developers are invited as limited collaborators
+- **The manifest’s default branch is protected**, preventing direct pushes without a pull request
```

Comment:
This also prevents overriding the history of the repository, so you can't sneak things in

Reply:
Ooh, good shout. Worth including.

```suggestion
- **The manifest’s default branch is protected**, preventing direct pushes without a pull request
- **The manifest’s commit history cannot be rewritten**, making it harder to sneak something in
```